css/stylish-portfolio.css

```diff
@@ -163,7 +163,14 @@ hr.small {
 }
 
 .header-text {
-    color: #ffffff
+    font-family: Convergence;
+    font-size: 25px;
+    font-style: normal;
+    font-variant: small-caps;
+    font-weight: bold;
+    line-height: 50px;
+    color: ffffff
```

Comment:
I missed this in my review.

We need a `;` at the end of color. Also hex needs to start with a `#`.

    color: #ffffff;